src/routes/category/Category.css

```diff
@@ -14,11 +14,22 @@
   font-weight: normal;
 }
 
-.categoryTopics {
-  margin: 0;
-  padding: 0;
+.featuredTopics {
```

Comment:
Why is this featured?

Reply:
In the spirit of keeping the same class names in different CSS files. It is the same class also.